src/opm/parser/eclipse/EclipseState/Grid/NNC.cpp

```diff
@@ -63,14 +63,6 @@ namespace Opm
         return result;
     }
 
-    void NNC::addNNC(const size_t cell1, const size_t cell2, const double trans) {
```

Comment:
Hi, I need this function to append more NNCs from numerical aquifers. Maybe I can add it back.

Reply:
OK - I will leave it in for now; do you have a work in progress branch for the numerical aquifer work which you could point me to?

Reply:
Yes. Most likely I am creating a WIP PR today.  The branch is https://github.com/GitPaean/opm-common/tree/numerical_aquifer